share/cmake/modules/CheckSSEFeatures.cmake

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+include(CheckCXXSourceCompiles)
+
+if (NOT CMAKE_SIZE_OF_VOID_P EQUAL 8)
+    if (MSVC)
+        set(CMAKE_REQUIRED_FLAGS "/arch:SSE2")
+    else (MSVC)
+        set(CMAKE_REQUIRED_FLAGS "-msse2")
+    endif (MSVC)
```

Comment:
Should MSVC be present in the else and endif?

Reply:
[It's optional](https://cmake.org/cmake/help/latest/command/if.html#command:if), I can remove it if that's the expected style for OpenColorIO.

Reply:
It's maybe a legacy syntax?  We don't do that elsewhere AFAIK, so my vote would be to remove it.